packages/ResizeDetector/README.md

```diff
@@ -0,0 +1,93 @@
+<!-- autogenerated don't modify -->
+
+# @paprika/resize-observer - 0.0.1
```

Comment:
When we publish we should remember to apply a `MINOR` bump, so we initially publish the `0.1.0` version.

cc @nahumzs

Reply:
you can put it on 0.1.0 I always remember whenever its a new component to select the option on lerna as custom version and I input `0.1.0` and since there its not any version with that number neither on github or npm will use it.